Reconstruction/main.cxx

```diff
@@ -49,6 +49,7 @@
 #include "vtkXMLStructuredGridWriter.h"
 #include "vtkXMLPolyDataWriter.h"
 #include "vtkMetaImageWriter.h"
+#include "vtkMathUtilities.h"
```

Comment:
Could you order the includes alphabetically please? It's better to avoid duplicates.